packages/jest-each/src/bind.ts

```diff
@@ -26,7 +25,7 @@ const PRETTY_PLACEHOLDER = '%p';
 const INDEX_PLACEHOLDER = '%#';
 
 export default (cb: Function, supportsDone: boolean = true) => (...args: any) =>
```

Comment:
```suggestion
 export default (cb: Function, supportsDone: boolean = true) => (...args: Array<unknown>) =>
```

or at least `Array<any>`